iina/QuickSettingViewController.swift

```diff
@@ -394,23 +404,19 @@ class QuickSettingViewController: NSViewController, NSTableViewDataSource, NSTab
     }
     let sliderValue = sender.doubleValue
     let value = AppData.minSpeed * pow((AppData.maxSpeed / AppData.minSpeed), sliderValue / sliderSteps)
```

Comment:
Looks like I left a spare set of parentheses here, would you mind removing them?